src/phantom.js

```diff
@@ -70,6 +78,14 @@ export default class Phantom {
     }
 
     /**
+     * Registers a callback to receive {String} messages from the Phantom context.
+     * @param {Function}
+     */
+    onMessage(cb) {
```

Comment:
It's interesting that you are suggesting any message here can be received. I feel like you are really just solving for your own problem. Would it make sense for someone who doesn't have context to use this?